src/Loader.php

```diff
@@ -325,9 +325,9 @@ protected function beginsWithAQuote($value)
     public function getEnvironmentVariable($name)
     {
         switch (true) {
-            case array_key_exists($name, $_ENV):
+            case isset($_ENV[$name]):
```

Comment:
The change in behaviour is now if the env variable was null, then the server variable gets incorrectly returned instead.